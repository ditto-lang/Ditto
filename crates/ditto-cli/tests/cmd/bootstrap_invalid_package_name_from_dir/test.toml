bin.name = "ditto"
args = ["bootstrap", "--no-make", "pkgs/not_quite"]
fs.sandbox = true
status = "failed"
