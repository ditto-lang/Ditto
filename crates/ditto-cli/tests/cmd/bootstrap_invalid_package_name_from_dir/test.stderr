error: "not_quite" cannot be used as a package name

Usage: ditto bootstrap [OPTIONS] <DIR>

For more information try '--help'
